### ISSUE TITLE
ctrl-alt mouse

### ISSUE THREAD

**Issue by scandum** [mintty/mintty#987]
I'd like to be able to user ctrl-alt for mouse events.

Would it be possible to make this a configurable setting? Or change the default to shift-alt and tie it in with the general shift-override?

**Comment by mintty** [comment]
I assume you refer to application mouse modes.
As ClickTargetMod=shift by default, Ctrl+Alt+mouse clicks are reported as events.
If you still have a problem, describe it clearly and provide an explicit test case, please.

**Comment by mintty** [comment]
OK, I checked about mouse scroll events only, Ctrl+Alt+click is in fact captured to open a popup menu.

**Comment by scandum** [comment]
My bad, I guess writing ctr+alt would have been clearer.

Changing it to shift+alt wouldn't work as that'd disable block selection in application mode.

If you could add some kind of config setting I'd appreciate it.

**Comment by mintty** [comment]
Removed the stray popup menu, and discovered this:
With Ctrl+Alt held, Windows does not send the usual mouse click message to the application (WM_RBUTTONDOWN) but uses its "non-client" variant (WM_NCRBUTTONDOWN) as if the click was on the window frame. @#§%$!!
But that can be caught and handled...

**Comment by mintty** [comment]
Uploaded a workaround, please test.

**Comment by scandum** [comment]
Getting a minor compilation error.

```
wininput.c: In function ‘show_menu_info’:
wininput.c:125:29: error: format ‘%X’ expects argument of type ‘unsigned int’, but argument 2 has type ‘DWORD’ {aka ‘long unsigned int’} [-Werror=format=]
  125 |   printf("menuinfo style %04X brush %p\n", mi.dwStyle, mi.hbrBack);
```

ctrl-alt mouse press works now, except that it doesn't generate a mouse release event upon release.

**Comment by mintty** [comment]
Uploaded fixes.

**Comment by scandum** [comment]
Works beautifully.

**Comment by mintty** [comment]
Released 3.1.5.
